pkg/api/conntrack_test.go

```diff
@@ -145,6 +145,58 @@ func TestConnTrackValidate(t *testing.T) {
 			},
 			conntrackInvalidError{unknownOutputRecord: true},
 		},
+		{
+			"Undefined selector key",
+			ConnTrack{
+				KeyDefinition: KeyDefinition{
+					FieldGroups: []FieldGroup{
+						{Name: "src", Fields: []string{"srcIP"}},
+					},
+				},
+				Scheduling: []ConnTrackSchedulingGroup{
+					{
+						Selector: map[string]string{
+							"srcIP":         "value",
+							"undefined_key": "0",
+						},
+					},
+				},
+			},
+			conntrackInvalidError{undefinedSelectorKey: true},
+		},
+		{
+			"Default selector on a scheduling group that is not the last scheduling group",
+			ConnTrack{
+				KeyDefinition: KeyDefinition{
+					FieldGroups: []FieldGroup{
+						{Name: "src", Fields: []string{"srcIP"}},
+					},
+				},
+				Scheduling: []ConnTrackSchedulingGroup{
+					{
+						Selector: map[string]string{},
+					},
+					{
+						Selector: map[string]string{
+							"srcIP": "value",
+						},
+					},
+				},
+			},
+			conntrackInvalidError{defaultGroupAndNotLast: true},
+		},
+		{
+			"Exactly 1 default selector",
```

Comment:
Do we try to test if 2 defaults are defined?

Reply:
I'll add a test for this